src/main/java/org/jenkinsci/plugins/gitclient/verifier/AbstractJGitHostKeyVerifier.java

```diff
@@ -1,64 +1,20 @@
 package org.jenkinsci.plugins.gitclient.verifier;
 
-import com.trilead.ssh2.Connection;
-import com.trilead.ssh2.KnownHosts;
-import com.trilead.ssh2.ServerHostKeyVerifier;
 import hudson.model.TaskListener;
-import java.io.IOException;
-import java.util.logging.Level;
-import java.util.logging.Logger;
+import org.eclipse.jgit.internal.transport.ssh.OpenSshConfigFile;
 import org.jenkinsci.remoting.SerializableOnlyOverRemoting;
 
-public abstract class AbstractJGitHostKeyVerifier implements ServerHostKeyVerifier, SerializableOnlyOverRemoting {
+public abstract class AbstractJGitHostKeyVerifier implements SerializableOnlyOverRemoting {
```

Comment:
what am I missing that this does not implement [org.apache.sshd.client.keyverifier.ServerKeyVerifier](https://javadoc.io/doc/org.apache.sshd/sshd-core/latest/org/apache/sshd/client/keyverifier/ServerKeyVerifier.html)